MIP40/MIP40c3-Subproposals/MIP40c3-SP10.md

```diff
@@ -6,7 +6,7 @@
 MIP40c3-SP#: 10
 Author(s): @wouter, @juanjuan
 Contributors:
-Tags: core-unit, cu-ses-001, active, budget
```

Comment:
Isn't this SP still active? their new one is just the MKR comp plan from my understanding.

Reply:
You're 100% right, of course. Fixing it right now